### ISSUE TITLE
tlg0661-textgroup Polyaenus should be tlg0616

### ISSUE THREAD

**Issue by AlisonBabeu** [OpenGreekAndLatin/First1KGreek#984]
Hi @sonofmun so I think we have dealt with a similar issue(#908) before. I wouldn't mind going with the "incorrect" TLG ID for Polyaenus, but I have already cataloged several editions of Polyaenus under the "correct" textgroup of tlg0616.

Should I follow the same procedure as outline in the issue I linked to above and simply delete these files on a branch, put them under tlg0616 as the textroup directory, and do a pull request. Thanks!

**Comment by sonofmun** [comment]
Yes, go ahead and do that. Make sure to add them to the appropriate `__cts__.xml` files as well.
